test/Microsoft.AspNetCore.Antiforgery.FunctionalTests/project.json

```diff
@@ -2,7 +2,8 @@
   "dependencies": {
     "AntiforgerySample": "1.0.0-*",
     "Microsoft.AspNetCore.TestHost": "1.0.0-*",
-    "Microsoft.AspNetCore.Testing": "1.0.0-*"
+    "Microsoft.AspNetCore.Testing": "1.0.0-*",
+    "Microsoft.NETCore.Platforms": "1.0.1-*"
```

Comment:
This code is checked in. Worth another commit or wait for next time we change this file?